tests/posix/clock/src/posix_clock.c

```diff
@@ -11,11 +11,17 @@
 
 void test_posix_clock(void)
 {
+	s32_t ret = 0;
 	s64_t nsecs_elapsed, secs_elapsed;
 	struct timespec ts, te;
 
 	printk("POSIX clock APIs\n");
-	clock_gettime(CLOCK_MONOTONIC, &ts);
+	ret = clock_gettime(CLOCK_REALTIME, &ts);
```

Comment:
Shouldn't this be two separate tests, one for CLOCK_REALTIME, and another for CLOCK_MONOTONIC?

Reply:
Hi @lpereira  This is done exclusively for getting code coverage. We do not support CLOCK_REALTIME so it will return with error (non 0 value). This will help to test condition if wrong argument has been passed with clock_gettime.